src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -557,7 +560,13 @@ class EmojiPickerMenu extends Component {
                     renderItem={this.renderItem}
                     keyExtractor={this.keyExtractor}
                     numColumns={CONST.EMOJI_NUM_PER_ROW}
-                    style={StyleUtils.getEmojiPickerListHeight(isFiltered, this.props.windowHeight)}
+                    style={[
+                        listStyle,
+                        // This prevents elastic scrolling when scroll reaches the start or end
+                        {overscrollBehaviorY: 'contain'},
+                        // Set overflow to hidden to prevent elastic scrolling when there is no enough contents to scroll in FlatList
```

Comment:
```suggestion
                        // Set overflow to hidden to prevent elastic scrolling when there are not enough contents to scroll in FlatList
```